src/2015-site-policies.rst

```diff
@@ -0,0 +1,81 @@
+Posting and commenting policy - Oct 2015
+########################################
+
+:author: C\. Titus Brown
+:tags: policy
+:date: 2015-10-19
+:slug: 2015-site-policies
+:category: misc
+
+This version of the site policy was posted in Oct 2015 and applies
+going forward from then.  I'll provide an updated link whenever I
+update the policy.
+
+The text was lifted from Captain Awkward's site policy at the URL
+http://captainawkward.com/site-policies-and-faqs/ on Oct 19, 2015, and
+then modified (significantly) from there.
+
+I'm open to suggestions for additions and removals; please post them
+in comments.  However, this set of policies stands until I change them -
+thanks for abiding by them!
+
+Copyright Notice
+----------------
+
+I own the copyright to all of my writing. Guest bloggers retain their
+own copyright. Quote all of us freely!  Link & attribute always!
+
+Questions
+---------
+
+By posting your comments to the blog you are granting me the right, in
+perpetuity, to use, quote, alter, and/or display those questions and
+comments however I see fit both for commercial and non-commercial
+distribution. If this concerns you, use a pseudonym and remove
+identifying details before you send, or simply post a link to your
+own blog.
+
+I do not share your personal email address or identity, with one
+exception: People who threaten or harass me or other posters will have
+their names, emails, and IP addresses shared with their internet
+service providers and, if necessary, law enforcement.
+
+Comments Policy
+---------------
+
+We moderate things in these here parts. Not every point of view
+deserves to be heard. Unmoderated spaces tend to be full of assholes.
```

Comment:
s/assholes/something-you-would-not-consider-impolite-if-directed-at-you-in-a-comment-on-your-blog/